packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -43,6 +44,7 @@
             <template #error>
                 <error-message
                     v-if="isAddressLocalityEmpty"
+                    data-error-message
```

Comment:
We should probably hook the `js` name to this name, just to make it clear as to the reason it's here (a javascript functionality)

`data-js-error-message`

Reply:
I didn't know we had that naming convention, but sounds good to me. Will amend the code 👍